api/src/uvisor-lib.c

```diff
@@ -37,5 +38,7 @@ int uvisor_lib_init(void)
         return -1;
     }
 
+    osRegisterForOsEvents(&uvisor_export_table->os_event_observer);
```

Comment:
Please add a comment that osEventObs won't allow a second call. For systems that don't make use of osRegisterForOsEvents we recommend to osRegisterForOsEvents(NULL) to disable further registrations (backdoor). Please make sure to submit a patch to osRegisterForOsEvents that implements and documents that. @AlessandroA @Patater @niklas-arm